scripts/install-docker.sh

```diff
@@ -12,3 +12,15 @@ if command -v amazon-linux-extras; then
 fi
 
 sudo yum install -y "docker-$DOCKER_VERSION" "containerd-$CONTAINERD_VERSION"
+
+# disable userland-proxy for docker
```

Comment:
Is this change applicable to AL2022? `install-docker.sh` is used in the AL2022 recipe [here](https://github.com/aws/amazon-ecs-ami/blob/main/al2022.pkr.hcl#L86) as well.

Reply:
Yes and @yinyic ran regression testing for AL2022 and tests succeeded(with couple of expected failures).